doc/reference/cmdline.rst

```diff
@@ -156,6 +156,16 @@ Database
     when creating new databases from the database-management screens, use the
     specified `template database`_. Defaults to ``template0``.
 
+.. option:: --db-maintenance <maintenance>
+
+   Odoo by default assumes that it can connect to the `postgres` database
```

Comment:
@LeartS mind if we reword / shorten this paragraph? The cmdline page is already quite dense and hard to read. I would suggest something like this:

```
.. option:: --db-maintenance <maintenance>

   For database operations that cannot be performed in the main
   database (e.g. centralized bus communications and database
   management operations), use the specified database instead.
   Defaults to ``postgres``, a database that is available by
   default in PostgreSQL installations.
```

PS: We should look into improving the look of `option::` /cc @xmo-odoo ;-)